spec/support/shared_examples/api_endpoint_with_authorization.rb

```diff
@@ -0,0 +1,12 @@
+# rubocop:disable RSpec/EmptyExampleGroup
+shared_examples "API endpoint with authorization" do
+  context "without authorization headers", document: false do
```

Comment:
`document: false` does not work on `context` as I know. It should be written like:

```
context "..." do
  example "...", document: false do
    ...
  end
end
```